doc/compiler/jitserver/Usage.md

```diff
@@ -20,14 +20,16 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-JITServer adds an additional two *personas* to the `java` executable: client mode and server mode. There are two new command line options that can be used to select a persona: `-XX:StartAsJITServer` and `-XX:+UseJITServer`.
+JITServer adds an additional two *personas* to the JVM: client mode and server mode. To start the JVM in server mode, run the `jitserver` launcher present under `bin` directory (along side `java` launcher). To start the JVM in client mode, there is a new command line option: `-XX:+UseJITServer`.
```

Comment:
Could you elaborate a bit more on how the `JITServer` launcher is built with the JVM? Something like `JITServer` launcher will be built only when `--enable-jitserver` is passed to the `bash configure` when building `OpenJ9`?

Reply:
I think this would be more appropriate in `Build.md`.